scalding-date/src/main/scala/com/twitter/scalding/DateOps.scala

```diff
@@ -59,8 +68,9 @@ object DateOps extends java.io.Serializable {
    * Return the guessed format for this datestring
    */
   private[scalding] def getFormatObject(s: String): Option[Format] = {
-    val formats: List[Format] = List(Format.DATE_WITH_DASH, Format.DATEHOUR_WITH_DASH,
-      Format.DATETIME_WITH_DASH, Format.DATETIME_HMS_WITH_DASH, Format.DATETIME_HMSM_WITH_DASH)
+    val formats: List[Format] = List(Format.DATE_WITHOUT_DASH, Format.DATE_WITH_DASH, Format.DATEHOUR_WITH_DASH,
```

Comment:
given the length of this list now i think it makes sense to do a 1 per line style

List(
 entry1,
 entry2,
 entry3
)